http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -136,6 +136,7 @@ class GceInterpreter[F[_]: Parallel: ContextShift](
             .newBuilder()
             .setDescription("Leonardo Managed Boot Disk")
             .setSourceImage(config.gceConfig.sourceImage.asString)
+            .setDiskSizeGb(bootDiskSize.gb.toString)
```

Comment:
Oops, I guess we weren't applying `bootDiskSize` in the request before?

Reply:
yea...it was removed in my first gpu PR because when we moved to using snapshot, it didn't seem to necessary so I removed it. But now that we're switching back to using custom image, we need to add it back.

I tried creating a new vm in dev, and bootDisk is still 50G...so I'm guessing maybe gcp is able to figure it out depending on the custom image size